aws/resource_aws_rds_cluster_test.go

```diff
@@ -2262,40 +2278,76 @@ resource "aws_rds_cluster" "test" {
 `, rName, engineMode)
 }
 
+func testAccAWSRDSClusterConfig_GlobalNetwork(rName string) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "azs" { }
+
+resource "aws_vpc" "vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+  	Name = "terraform-acctest-rds-cluster-global-cross-region"
```

Comment:
Setting up cross-region acceptance testing is a pretty complicated undertaking (see also: `TestAccAWSRDSCluster_EncryptedCrossRegionReplication`). 😅 Since the implementation in 9b7a9b0d0861b46d84ad10d35de32950a6142822 is only halfway there and has a few other little issues, opting to merge this pull request, but revert that commit until we can get a full, new test setup, preferably separate from the existing ones.